examples/sprite_camera_follow/main.rs

```diff
@@ -0,0 +1,184 @@
+extern crate amethyst;
+
+use amethyst::{
+    assets::{AssetStorage, Loader},
+    core::{Parent, Transform, TransformBundle},
+    ecs::{Component, Entity, Join, NullStorage, Read, ReadStorage, System, WriteStorage},
+    input::{InputBundle, InputHandler},
+    prelude::*,
+    renderer::{
+        Camera, ColorMask, DepthMode, DisplayConfig, DrawSprite, MaterialTextureSet, Pipeline,
+        PngFormat, Projection, RenderBundle, SpriteRender, SpriteSheet, SpriteSheetFormat,
+        SpriteSheetHandle, Stage, Texture, TextureMetadata, Transparent, ALPHA,
+    },
+    utils::application_root_dir,
+};
+
+#[derive(Default)]
+struct Player;
+
+impl Component for Player {
+    type Storage = NullStorage<Self>;
+}
+
+struct MovementSystem;
+
+impl<'s> System<'s> for MovementSystem {
+    type SystemData = (
+        ReadStorage<'s, Player>,
+        WriteStorage<'s, Transform>,
+        Read<'s, InputHandler<String, String>>,
+    );
+
+    fn run(&mut self, (players, mut transforms, input): Self::SystemData) {
+        let x_move = input.axis_value("entity_x").unwrap();
+        let y_move = input.axis_value("entity_y").unwrap();
+
+        for (_, transform) in (&players, &mut transforms).join() {
+            transform.translation.x += x_move as f32 * 5.0;
+            transform.translation.y += y_move as f32 * 5.0;
+        }
+    }
+}
+
+fn load_sprite_sheet(world: &mut World, png_path: &str, ron_path: &str) -> SpriteSheetHandle {
```

Comment:
This thing really need to get merged into the engine. I keep seeing it copy pasted everywhere... 
Feel free to not do it for this pr, but it would be nice if someone could do it at some point.

Reply:
It can't be done in a very nice way as it requires the user provides a texture ID. But it is going away thanks to Rhuagh's work, we will slightly modify it when we will redesign it.

Reply:
Just add a `SpriteSheetLoader` helper that abstract this shit away tbh.